ui/src/components/data-agent/ConnectorConfiguration.tsx

```
@@ -0,0 +1,54 @@
+import { FC } from 'react';
+import { motion } from 'framer-motion';
+
+interface ConnectorConfigurationProps {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)